google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -206,12 +200,15 @@ def _get_or_create_sequencer(self, topic: str, ordering_key: str) -> SequencerTy
         sequencer_key = (topic, ordering_key)
         sequencer = self._sequencers.get(sequencer_key)
         if sequencer is None:
+            # Disable pytype false positive...
+            # pytype: disable=wrong-arg-types
```

Comment:
Can we just remove `pytype` altogether?

Reply:
In fact, we can, and will do it. Can't really see much value compared to `mypy`.

I will make the changes in https://github.com/googleapis/python-pubsub/pull/551, since I'd like to keep the class refactoring part separate (for an easier rollback, should that be needed).